### ISSUE TITLE
Classifier pretraining question

### ISSUE THREAD

**Issue by chaoyan1037** [lwzeng/GC-Net#1]
Hi Weizeng,

Thanks for sharing your implementation! As I read your paper, I have a question about the classifier training.
I notice that the classifier takes the masked images as input during inference. So do you pre-train the classifier on masked or unmasked images?

**Comment by lwzeng** [comment]
Hi, We pre-train the classifier with the original images. You can find the pre-train code of the classifier on classifier/classifier_google.py or classifier/classifier_vgg.py. Another thing to note is that when training the detector, the classifier takes the masked images as input. We multiply the mask by the image normalized by the mean and variance, Instead of normalizing after multiplying. This can be seen in the training code of our detector.

**Comment by chaoyan1037** [comment]
Thanks for the response!
